app/controllers/government_rulesets_controller.rb

```diff
@@ -4,17 +4,39 @@ class GovernmentRulesetsController < ApplicationController
   before_action :clear_errors
 
   def show
-    upload = Upload.find(session[:upload_id])
-    return redirect_to root_path, alert: "Please re upload your file" if upload.nil?
+    @upload = Upload.find(session[:upload_id])
+    return redirect_to root_path, alert: "Please re-upload your file" if upload.nil?
 
-    @spectral_results =
-      JSON.parse(
-        Linters::Spectral.new(upload:).lint_to_json
-      )
+    @issues = []
+    @score = 100
+    @counter = 0
+    spectral_hash = JSON.parse(spectral_output)
```

Comment:
[suggestion]
I know this is pedantic, but it did take me a double take to work out what is going on ....

`JSON.parse(spectral_output)` returns an array of hashes, so the label `spectral_hash` is misleading as it suggest it contains a single hash. I'd suggest `spectral_hashes` or even `issues`

Reply:
Yes this is a left over from a previous iteration - I agree "issues" is easier to follow. Changed.